binary/private/binary_mdot.f90

```diff
@@ -59,7 +59,6 @@ integer function check_implicit_rlo(binary_id, new_mdot)
          end if
          s => b% s_donor
          use_sum = .false.
-         detachment = .false.
```

Comment:
@orlox @matthiasfabry (wasn't sure if tagging was necessary)

I'm hunting for a bit-for-bit issue that was probably introduced in this PR. Should this line have been replaced with `detached = .false.`, rather than simply deleted? I'm working my way through the procedure calls to see if `detached` is always set before it's referred to.

Reply:
Great! I'll let you make whatever changes you have lined up then rerun on `ifort` here.